src/coreclr/jit/codegenarm64.cpp

```diff
@@ -5433,7 +5433,7 @@ void CodeGen::genProfilingLeaveCallback(unsigned helper)
 
 // Uncomment "#define ALL_ARM64_EMITTER_UNIT_TESTS" and one or more of the categories to run all the unit tests here.
 // After adding a unit test, and verifying it works, put it under this #ifdef, so we don't see it run every time.
-#define ALL_ARM64_EMITTER_UNIT_TESTS
+// #define ALL_ARM64_EMITTER_UNIT_TESTS
```

Comment:
seems this was left from https://github.com/dotnet/runtime/pull/95127

Reply:
It would be nicer if this was a debug config option, as it's a pain to uncomment in order to test. But understood why you might not want that.

Reply:
@a74nh A debug config option would be fine. Either several (one per "category"), or maybe one string that takes a (comma? semicolon? space?) separated list of categories and generates/runs each.

It is as it is for no particularly good reason, except simplicity. And the fact this code is written and normally used only once or twice, so too much engineering doesn't seem worth it.

Reply:
>debug config option

This was done in https://github.com/dotnet/runtime/pull/96005. This PR will need rebasing as that PR also moved the tests to a new file.

Reply:
Rebased and moved the tests to `codegenarm64test.cpp`.